src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -23,214 +25,189 @@ private static byte[] GetJsonBytes<T>(T elem)
             return Encoding.UTF8.GetBytes(elem.ToString());
         }
 
-        [Benchmark]
-        public byte GetByte()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
+        public void GetBoolean()
         {
-            byte result = 0;
-            var reader = new Utf8JsonReader(_jsonIntegerNumberBytes);
+            var reader = new Utf8JsonReader(_jsonFalseBytes);
             reader.Read();
 
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < OperationsPerBenchmark; i++)
             {
-                result += reader.GetByte();
+                reader.GetBoolean();
             }
-            return result;
         }
 
-        [Benchmark]
-        public sbyte GetSByte()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
+        public void GetByte()
         {
-            sbyte result = 0;
             var reader = new Utf8JsonReader(_jsonIntegerNumberBytes);
             reader.Read();
 
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < OperationsPerBenchmark; i++)
             {
-                result += reader.GetSByte();
+                reader.GetByte();
             }
-            return result;
         }
 
-        [Benchmark]
-        public short GetInt16()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
+        public void GetSByte()
         {
-            short result = 0;
             var reader = new Utf8JsonReader(_jsonIntegerNumberBytes);
             reader.Read();
 
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < OperationsPerBenchmark; i++)
             {
-                result += reader.GetInt16();
+                reader.GetSByte();
             }
-            return result;
         }
 
-        [Benchmark]
-        public int GetInt32()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
```

Comment:
Overall it's a good idea to use `OperationsPerInvoke` for such benchmarks, but in case of an existing benchmark we can't do it. It would suddenly change the time reported by BDN (`newTime = oldTime / OperationsPerBenchmark`) which is stored in the [ReportingSystem](https://pvscmdupload.blob.core.windows.net/reports/allTestHistory%2frefs%2fheads%2fmain_x64_Windows%2010.0.18362%2fSystem.Text.Json.Tests.Perf_Get.GetInt16.html) and would be recognized by the automation as a sudden perf improvement. 

We could do that if we were also about to change the benchmark ID (namespace.typeName.MethodName), but this would be done at a price of loosing the historical data in the Reporting System (it would be recognized as a new benchmark).

```suggestion
        [Benchmark]
```

That is why we hardly ever rename the benchmarks. For more detail please refer to:

https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#benchmarks-are-immutable

Reply:
I imagined this might be a concern, but my assumption was that it wouldn't be an issue provided that it's an improvement versus a regression.